scripts/smoke-process-lifecycle.ps1

```
@@ -0,0 +1,687 @@
+    [string]$CandidateBinary,
+
+    [string]$EvidencePath = "",
+
+    [string]$RunDir = "",
+
+    [int]$TimeoutSeconds = 60
+)
+
+$ErrorActionPreference = "Stop"
+$ParallelSessions = 8
+$RepoRoot = Split-Path -Parent $PSScriptRoot
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'RepoRoot' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)